var/spack/repos/builtin/packages/openjdk/package.py

```diff
@@ -27,7 +27,7 @@
             "7d3ab0e8eba95bd682cfda8041c6cb6fa21e09d0d9131316fd7c96c78969de31",
             "https://github.com/adoptium/temurin21-binaries/releases/download/jdk-21.0.3%2B9/OpenJDK21U-jdk_aarch64_linux_hotspot_21.0.3_9.tar.gz",
         ),
-        "Darwin-aarch64": (
+        "Darwin-arm64": (
```

Comment:
I know I'm late with this, but can you document in a comment on the pr why this change was necessary? Did the os name get changed or was this wrong before?

Reply:
Nevermind, I see the link to https://github.com/spack/spack/pull/30826 now...